pavelib/tests.py

```diff
@@ -212,6 +212,26 @@ def coverage(options):
     call_task('diff_coverage', options=dict(options))
 
 
+@no_help
+@task
+@needs('pavelib.prereqs.install_prereqs')
+def combine_jenkins_coverage():
+    """
+    Combine coverage reports.
```

Comment:
I say just add some doc here pointing out that these is for combining coverage reports in Jenkins specifically.